src/main/java/domain/Rank.java

```diff
@@ -0,0 +1,39 @@
+package domain;
+
+import java.math.BigDecimal;
+import java.util.Arrays;
+
+public enum Rank {
+    FIRST(6, BigDecimal.valueOf(2000000000)),
+    SECOND(5, BigDecimal.valueOf(1500000)),
+    THIRD(4, BigDecimal.valueOf(50000)),
+    FOURTH(3, BigDecimal.valueOf(5000)),
```

Comment:
```suggestion
    FOURTH(3, BigDecimal.valueOf(5_000)),
```

돈과 같이 1000단위로 구분은 다음과 같이 underbar(_)를 활용해 구현하면 코드 가독성이 좋아집니다.